test/parallel/test-fs-watchfile.js

```diff
@@ -63,3 +63,25 @@ fs.watchFile(enoentFile, {interval: 0}, common.mustCall(function(curr, prev) {
     fs.unwatchFile(enoentFile);
   }
 }, 2));
+
+// Watch events should callback with a filename
+if (common.isLinux || common.isOSX) {
```

Comment:
I would add Windows too. You can also add AIX according to @gireeshpunathil's suggestion.